integration-test/gradlew

```diff
@@ -0,0 +1,234 @@
+#!/bin/sh
```

Comment:
Please also add Gradle wrapper, so it can be run locally without Gradle 7.3 installed in the system